RLTsquare/CustomRoute/etc/frontend/di.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Magento\Framework\App\RouterList">
+        <arguments>
+            <argument name="routerList" xsi:type="array">
+                <item name="customRoute" xsi:type="array">
```

Comment:
Naming convention - we should not use "customRoute"... we should adjust the naming properly to match our module.